tools/MultiPEN/MultiPEN-feature-selection.xml

```diff
@@ -7,24 +7,20 @@
         <exit_code range="1:" level="fatal" />
     </stdio>
     <command><![CDATA[
-run_MultiPEN_slurm.sh FeatureSelection ./ '$ExpressionData' '$Interactions' '$SampleClass' $lambda $DecisionThreshold
-#if str($MaxIter)
-    $MaxIter
-#end if
+run_MultiPEN_slurm.sh FeatureSelection ./ '$ExpressionData' '$Interactions' '$SampleClass' $lambda $optionalParameters
 &&
-mv MultiPEN-performance_feature-selection_lambda${lambda}.txt Performance.txt &&
-mv MultiPEN-Rankings_lambda${lambda}.txt Rankings.txt &&
-mv MultiPEN-vts_lambda${lambda}.txt vts.txt &&
-mv MultiPEN-Rankings_lambda${lambda}_genes-higher-in-cases.txt Rankings-higherInCases.txt &&
-mv MultiPEN-Rankings_lambda${lambda}_genes-higher-in-control.txt Rankings-higherInControl.txt
+mv MultiPEN-performance_feature-selection_lambda${lambda}_${optionalParameters}.txt Performance.txt &&
+mv MultiPEN-Rankings_lambda${lambda}_${optionalParameters}.txt Rankings.txt &&
+mv MultiPEN-vts_lambda${lambda}_${optionalParameters}.txt vts.txt &&
+mv MultiPEN-Rankings_lambda${lambda}_genes-higher-in-cases_${optionalParameters}.txt Rankings-higherInCases.txt &&
+mv MultiPEN-Rankings_lambda${lambda}_genes-higher-in-control_${optionalParameters}.txt Rankings-higherInControl.txt
```

Comment:
Add single quotes around all the `MultiPEN-....txt' filenames (again because `${optionalParameters}` is a text param)